flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/Executor.java

```diff
@@ -42,54 +42,50 @@ public interface Executor {
      * Open a new session by using the given session id.
      *
      * @param sessionId session identifier.
-     * @return used session identifier to track the session.
      * @throws SqlExecutionException if any error happen
      */
-    String openSession(@Nullable String sessionId) throws SqlExecutionException;
+    void openSession(@Nullable String sessionId) throws SqlExecutionException;
 
     /**
      * Close the resources of session for given session id.
      *
-     * @param sessionId session identifier
      * @throws SqlExecutionException if any error happen
      */
-    void closeSession(String sessionId) throws SqlExecutionException;
+    void closeSession() throws SqlExecutionException;
 
     /**
      * Returns a copy of {@link Map} of all session configurations that are defined by the executor
      * and the session.
      *
-     * <p>Both this method and {@link #getSessionConfig(String)} return the same configuration set,
-     * but different return type.
+     * <p>Both this method and {@link #getSessionConfig()} return the same configuration set, but
+     * different return type.
      */
-    Map<String, String> getSessionConfigMap(String sessionId) throws SqlExecutionException;
```

Comment:
I have one question. Original  `LocalExecutor` implements `Executor` have a map `ConcurrentHashMap<String, SessionContext> contextMap`, but now, we just delete it. Does this mean that each executor can only have one session? (If that, why we need sessionId, we can just define an executorId instead ? ). Also, I think if you delete this `contextMap`, you need add tests to verify we don't need this map. WDYT ？

Reply:
> Does this mean that each executor can only have one session?
Yes, you are right. Executor only works for one session in the new design. The Gateway is designed for the multitenancy.

> why we need sessionId, we can just define an executorId instead?
I think session id is still useful. For example, SQL Client can convey the user name/password with the current API. So I am prone to keep it unchanged and prepare for the future authentication mechanism.

Reply:
Ok, I got it.